src/lib/utils/thorchain/hooks/useGetThorchainSaversDepositQuoteQuery.tsx

```diff
@@ -1,22 +1,24 @@
 import type { Asset } from '@shapeshiftoss/types'
-import { useQuery } from '@tanstack/react-query'
+import { skipToken, useQuery } from '@tanstack/react-query'
 import { useMemo } from 'react'
 import { type BigNumber, bnOrZero } from 'lib/bignumber/bignumber'
+import type { PartialFields } from 'lib/types'
+import type { ThorchainSaversDepositQuoteResponseSuccess } from 'state/slices/opportunitiesSlice/resolvers/thorchainsavers/types'
 import { getMaybeThorchainSaversDepositQuote } from 'state/slices/opportunitiesSlice/resolvers/thorchainsavers/utils'
 
+type FetchThorchainDepositQuoteInput = {
+  asset: Asset
+  amountCryptoBaseUnit: BigNumber.Value | null | undefined
+}
+
 export type GetThorchainSaversDepositQuoteQueryKey = [
   'thorchainSaversDepositQuote',
-  {
-    asset: Asset
-    amountCryptoBaseUnit: BigNumber.Value | null | undefined
-  },
+  PartialFields<FetchThorchainDepositQuoteInput, 'asset'>,
```

Comment:
This seems to work as expected, but should we have `amountCryptoBaseUnit` also a part of the query key?

Reply:
We do! `PartialFields` makes some fields as optional, while keeping others as they were (optional or required) so this is effectively 
```typescript
type FetchThorchainDepositQuoteInput = {
  asset: Asset
  amountCryptoBaseUnit?: BigNumber.Value | null | undefined
}
```

Reply:
Oh, right you are ser!